zilliqa/src/cfg.rs

```diff
@@ -45,6 +45,8 @@ pub struct NodeConfig {
     pub tx_retries: u64,
     /// The maximum duration between a recieved block's timestamp and the current time. Defaults to 10 seconds.
     pub allowed_timestamp_skew: Duration,
+    /// Whether to allow a single node to generate its own chain.
+    pub allow_single_node_network: bool,
```

Comment:
IMO there should be no reason a network with a single node doesn't just work out of the box without any configuration.

Reply:
I agree, although as it currently stands, it can be that the network will appear to work but in fact it is just running with the one validator and the others fail to sync/join the committee. Having it explicitly set whether you allow only single validator avoids this. If you want I can take it out since it's not likely to happen the way you use it (start everyone at same instant)